ui/ducks/domains.js

```diff
@@ -167,18 +172,20 @@ export function initializeDomainSlice() {
   return (dispatch, getState) => {
     const state = getState();
     const chainId = getCurrentChainId(state);
-    const networkName = CHAIN_ID_TO_ETHERS_NETWORK_NAME_MAP[chainId];
-    const chainIdInt = parseInt(chainId, 16);
-    const ensAddress = ensNetworkMap[chainIdInt.toString()];
-    const networkIsSupported = Boolean(ensAddress);
-    if (networkIsSupported) {
-      web3Provider = new Web3Provider(global.ethereumProvider, {
-        chainId: chainIdInt,
-        name: networkName,
-        ensAddress,
-      });
-    } else {
-      web3Provider = null;
+    if (!useSnapForENS) {
```

Comment:
This block will disappear once https://github.com/MetaMask/snaps/issues/2621 gets fixed and `endowment:name-lookup` is no longer experimental